app/views/layouts/_google_analytics.html.erb

```diff
@@ -0,0 +1,9 @@
+<% if Rails.env.production? ||  Rails.env.development? %>
```

Comment:
Do we really want to enable this in development? Yes while initially testing it, but once we're happy it's working - production only I think.

Reply:
Actually, would it be even easier just to use the presence of the environment variable?
```erb
<% if ENV['GOOGLE_ANALYTICS_TRACKING_ID'].present? %>
```

Reply:
Done. Thanks, good suggestion.